components/expenses/MarkExpenseAsPaidBtn.js

```diff
@@ -25,6 +24,7 @@ class MarkExpenseAsPaidBtn extends React.Component {
     mutate: PropTypes.func,
     refetch: PropTypes.func,
     intl: PropTypes.object.isRequired,
+    errorMessage: PropTypes.func,
```

Comment:
`errorMessage` is not a very explicit name, from it I would expect the prop to be a string.

The role of this prop is to provide a callback when there's an error, so `onError` seems appropriate.

Reply:
You're right, that name isn't very helpful. `onError` seems more appropriate.